bootloader/CMakeLists.txt

```
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.19.2)
+
+# Disable CMake's compiler test for cross-compilation
+set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
+
+# Specify the cross-compiler
+set(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_C_COMPILER arm-none-eabi-gcc -nostdlib)
+set(CMAKE_CXX_COMPILER arm-none-eabi-g++)
+set(CMAKE_ASM_COMPILER arm-none-eabi-gcc)
+
+# Project name
+project(Bootloader C CXX ASM)
```

```suggestion
project(Bootloader C ASM)
```
I don't know if this will work but better avoid evil C++ code